Sources_v3/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -318,3 +343,7 @@ extension ClientError {
         }
     }
 }
+
+extension Int {
+    static let maxNumberOfLatestReactions = 10
+}
```

Comment:
IMHO we should limit extending standard types to a minimum, and avoid where unnecessary

Reply:
Ok, I see your point. Let's have the constant in **MemberDTO** then: 
```swift
private static let maxNumberOfLatestReactions = 10
```

Reply:
resolved in 8ac7bb42aac85e9e420e5cfa0450d6536bfbda46